File: SFA-Client/src/app/_models/salesRepItemBatch.ts
export class SalesRepItemBatch {
  id: number;
  userId: number;
  salesRepName: string;
  itemBatchId: number;
  itemBatchName: string;
  itemCount: number;
  storeId: number;
  storeName: number;

  constructor(salesRepItemBatch) {
    this.id = salesRepItemBatch.id;
    this.userId = salesRepItemBatch.userId;
    this.salesRepName = salesRepItemBatch.salesRepName;
    this.itemBatchId = salesRepItemBatch.itemBatchId;
    this.itemBatchName = salesRepItemBatch.itemBatchName;
    this.itemCount = salesRepItemBatch.itemCount;
    this.storeId = salesRepItemBatch.storeId;
    this.storeName = salesRepItemBatch.storeName;
  }
}

File: SFA-API/Domain/SalesRepItemBatch.cs
using System.Collections.Generic;

namespace Domain
{
    public class SalesRepItemBatch
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int ItemBatchId { get; set; }
        public int ItemCount { get; set; }
        public int StoreId { get; set; }
        public virtual UserDetail UserDetail { get; set; }
        public virtual ItemBatch ItemBatch { get; set; }
    }
}